File: LEVEL1/Function&Array/Span of array.java
import java.io.*;
import java.util.*;

public class Main{

public static void main(String[] args) throws Exception {
    Scanner scn = new Scanner(System.in);
    int n = scn.nextInt();
    int[] arr = new int[n];

    for(int i = 0; i<n; i++){
        arr[i] = scn.nextInt();
    }

    int max = arr[0];
    for(int i = 1; i<n; i++){
        if(max < arr[i]){
            max = arr[i];
        }
    }

    int min = arr[0];
    for(int i = 1; i<n; i++){
        if(min > arr[i]){
            min = arr[i];
        }
    }

    int sum = max - min;
    System.out.println(sum);
 }

}

File: DSAJS/Array&Functions/spanOfArray.js
function spanOfArray(nums) {
  let min = nums[0], max = nums[0];
  for(let i in nums) {
    if(min > nums[i]) {
      min = nums[i];
    }
    if(max < nums[i]) {
      max = nums[i];
    }
  }
  return max - min;
}

const nums = [5, 3, 6, 1, 7];
console.log(spanOfArray(nums));

// T.C O(n)
// S.C O(1)